src/uikit.cs

```diff
@@ -863,6 +863,11 @@ partial interface NSTextAttachment : NSTextAttachmentContainer, NSCoding
 		[NullAllowed] // by default this property is null
 		[Export ("fileWrapper", ArgumentSemantic.Retain)]
 		NSFileWrapper FileWrapper { get; set; }
+
+		[Watch (6,0), TV (13,0), iOS (13,0)]
+		[Static]
+		[Export ("textAttachmentWithImage:")]
+		NSTextAttachment FromImage (UIImage image);
```

Comment:
No verb in method name.

Reply:
We have other instances of from usage

* NSLayoutConstraint [] FromVisualFormat
* UIApplicationShortcutIcon FromType
* UIApplicationShortcutIcon FromTemplateImageName
* UIApplicationShortcutIcon FromContact
* UIColor FromWhiteAlpha
* UIColor FromCGColor

and many others, I think this one is fine, any `From` usage is fine in my opinion as long as it is a Static method and returns the container type.